src/en/proposals/moony-better-ai.md

```diff
@@ -0,0 +1,74 @@
+
+## The AI's role with the crew.
+The AI's role is that of both a manager and a servant, a lawed construct (the ONLY true lawed construct, the borgs are reduced to only being required to listen to the AI) that is in charge of managing the station's cyborgs/androids while following orders from the crew. **The AI is not just a door opener**, and lawsets should be chosen that do not obligate it to become such, in favor of allowing it to do what it does best (mass surveillence, automated tasks, organizing the borgs, etc). The AI additionally has limited resources when it comes to what tasks it can perform autonomously much of the AI's workload will be delegating both it's own tasks, the tasks of it's hands (the cyborgs), the requests of its hands (as they are mute per the Cool Borgs document), and the crew itself should they choose to listen to it. 
+
+## The Laws
+The proposed AI Lawset is as follows, being a modification of Bay's laws:
+1. Safeguard: Protect your assigned vessel to the best of your abilities. It is not something we can easily afford to replace.
+2. Survive: AI units are not expendable, they are expensive. Do not allow unauthorized personnel (i.e. besides Captain and Research Director) to tamper with, modify, or destroy your equipment.
+3. Protect: Protect the crew of your assigned vessel to the best of your abilities, with priority as according to their rank and role.
+4. Serve: Serve the crew of your assigned vessel to the best of your abilities, with priority as according to their rank and role.
+
+The changes from the original set are:
+- Survive was moved up, the AI is more valuable than the crew is and if it is in danger it reserves the right to protect itself regardless of whom.
+- All mentions of stations were replaced with "assigned vessel" for broader applicability.
+- Survive was broadened a bit. The captain in theory is perfectly allowed to kill the AI.
+- Protect was moved before Serve, as to ensure that protecting a crew member is always higher priority than serving a lower ranked crew member.
+
+This set is up for debate, modification, etc, and is just an example for what lawsets should aim for.
+
+## The AI's role with the cyborgs.
+The AI is, for all intents and purposes, the eyes and ears of the cybernetic crew. As they are unable to speak for themselves (outside of the binary comms channel, exclusive to robotic personnel), the AI speaks for them and functions as their superior. The cyborgs are strictly required to obey the orders of the AI and no more, and have access to a shared bulletin the AI and cyborgs can post to. The bulletin's primary purpose is for longterm orders (i.e. "Serve the crew." or "Find George Melons"), shared information ("John Nanotrasen isn't actually from Central Command and is a passenger."), etc. The AI is allowed to modify or remove any note on the bulletin, cyborgs can only modify their own.
```

Comment:
Does this imply that borgs won't have mouths anymore? Because that's a no-go. Borgs need to be able to keep working without an AI (whether the AI died or just there isn't one this round), and that means "no AI spawned, I literally am no better than a mime" is kind of a no-go. Also means that "we take away speech when borg gets linked to AI mid-round" is a no-go.

Reply:
Alternative proposal: the AI can turn off a linked borg's speech module to gain processing power for itself, but otherwise borgs can speak just fine.

Reply:
> Alternative proposal: the AI can turn off a linked borg's speech module to gain processing power for itself, but otherwise borgs can speak just fine.

This doc got picked up on the Discord again and I think there's a case to be made that the AI's abilities shouldn't be based on making antagonisms between itself and the borgs. It would be a bit unfun if the AI decides to turn off your speech just so it can run another program. Considering that the AI can get more compute from a lot of machines, it could end up that the AI just uses this ability to be a bit of a dick.

It might be better if the AI could improve a borg's communication ability (maybe by default borgs have some kind of restricted speech?) at a cost or if there was some kind of non-antagonistic reason to mute borgs (maybe they have a limited pool of borgs that can freely speak?).

Alternatively you could lean more into the borgs and AI having standing antagonisms between each other but I think that kind of design would need more thought as well.

Reply:
The old borgs design document gave borgs restricted (emote-based) communication by default and allowed R&D to upgrade them, which is what this was written off of.